src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs

```diff
@@ -279,6 +296,30 @@ public void StartTestRun(TestRunCriteriaWithSources runCriteria, ITestRunEventsH
             this.onMessageReceived = (sender, args) => this.OnExecutionMessageReceived(sender, args, eventHandler);
             this.channel.MessageReceived += this.onMessageReceived;
 
+            // This code section is needed because we altered the old testhost launch process for
+            // the debugging workflow. Now we don't ask VS to launch and attach to the testhost
+            // process for us as we previously did, instead we launch it as a standalone process
+            // and rely on the testhost to ask VS to attach the debugger to itself.
```

Comment:
Its great that we have comments in code. Can we also have an RFC for this change if we haven't already? It'd be great to have something to point folks (extension authors) to so they can understand the details.

Reply:
An [RFC exists](https://github.com/microsoft/vstest-docs/pull/216/files?short_path=fb0ba30#diff-fb0ba30e09566fc43521c71dcf5fb205) - although it probably needs some updates around the issues addressed in the final commits.

Reply:
Cool, we'll probably want to touch on how other IDEs would be affected there too. Can we also update [this](https://github.com/microsoft/vstest-docs/blob/master/RFCs/0003-Test-Execution-Protocol.md#new-flow-for-debug-testscustom-test-host-process) to mention the new design?

Reply:
Opened a new issue [here](https://github.com/microsoft/vstest-docs/issues/232) to keep track of the RFC update and assigned it to myself.